app/views/media_library/show.html.erb

```diff
@@ -8,9 +8,21 @@
                 when MediaContent::TYPE_PHOTO
                   is_vertical = @media_content.photo_sizes.where(label: PhotoSize::LARGE).first.is_vertical
                   @media_content.photo_sizes.where(label: PhotoSize::LARGE).first.try(:url)
+                when MediaContent::TYPE_COLLECTION
+                  photo = @media_content.graphics.first
+                  is_vertical = photo.photo_sizes.where(label: PhotoSize::LARGE).first.is_vertical
+                  photo && photo.photo_sizes.where(label: PhotoSize::LARGE).first.try(:url)
+                when MediaContent::TYPE_GRAPHIC
```

Comment:
@simaob how can I put the `TYPE_PHOTO` & `TYPE_GRAPHIC` in the same condition? I was try with `MediaContent::TYPE_PHOTO || MediaContent::TYPE_GRAPHIC` but doesn't work.